frontend/src/Editor/Container.jsx

```diff
@@ -335,7 +335,7 @@ export const Container = ({
     }
   }
 
-  React.useEffect(() => {}, [selectedComponents]);
+  React.useEffect(() => { }, [selectedComponents]);
```

Comment:
@LalitChauhan27 Can you revert this change? It might throw an error on lint
```suggestion
  React.useEffect(() => {}, [selectedComponents]);
```